doctr/models/detection/differentiable_binarization.py

```diff
@@ -162,17 +162,18 @@ def bitmap_to_boxes(
 
     def __call__(
         self,
-        x: tf.Tensor,
+        x: Dict[str, tf.Tensor],
     ) -> List[np.ndarray]:
         """Performs postprocessing for a list of model outputs
 
         Args:
-            x: raw output of the model, of shape (N, H, W, 1)
+            x: dictionary of the model output
```

Comment:
I suggest adding back the shape (specifying which key needs to be filled with which shape for the method to work)